styleguide/source/_patterns/02-molecules/article-preview.twig

```diff
@@ -0,0 +1,56 @@
+{% set link = topic_article_preview.title %}
+{% set class = topic_article_preview.class %}
+
+{#Topic videos#}
+{% macro topic_video(topic_article_video) %}
+  {% set video = topic_article_video %}
+  {% if video %}
+    {% include "@atoms/media/video.twig" %}
+  {% endif %}
+{% endmacro %}
+
+{#Topic images#}
+{% macro topic_image(topic_article_image) %}
+  {% set image = topic_article_image %}
+  {% if image %}
+    <div class="ama__topic_article-preview-container">
+      {% include "@atoms/media/image.twig"  with { 'class': 'ama__topic_article-preview-container-image'} %}
+    </div>
+  {% endif %}
+{% endmacro %}
+
+{#Topic related #}
+{% macro topic_description(topic_article_preview) %}
+  {% set title = topic_article_preview.title_only %}
+  {% set related = topic_article_preview.related %}
+  {% set link = topic_article_preview.link %}
+  {% set paragraph = topic_article_preview.paragraph %}
+
+  {% if paragraph and not title and not related %}
+    <div class="ama__topic_article-preview_description">
+      {% include "@atoms/paragraph.twig" %}
+    </div>
+    <div class="ama__metadata topic_article-preview_metadata col-xs-12">
+      {% include "@molecules/article-metadata.twig" %}
+    </div>
+  {% endif %}
+{% endmacro %}
+
+{#Show only title#}
+{% macro topic_title(topic_article_title) %}
+  {% set link = topic_article_title %}
+  {% if link  %}
+   <h2 class="ama__topic_article-preview_container-title">
+      {% include "@atoms/link.twig" %}
```

Comment:
This is just showing as `@atoms/link.twig`. It looks like the nesting folders are still there under atoms, so that is likely what is causing the issue here.